Nasal/c172p.nas

```diff
@@ -12,6 +12,9 @@ var autostart = func (msg=1) {
     # Reset battery charge and circuit breakers
     electrical.reset_battery_and_circuit_breakers();
 
+	# Repair engine damage
+	setprop("/fdm/jsbsim/engine/damage-level", 0);
```

Comment:
This should be moved to `reset_all_damage` function in `Nasal/physics.nas`

Reply:
Done. Could you remove this to remove the thing that says you are still requesting changes?